stat.c

```diff
@@ -486,6 +489,25 @@ static void show_ddir_status(struct group_run_stats *rs, struct thread_stat *ts,
 					samples,
 					ts->percentile_list,
 					ts->percentile_precision, name, out);
+
+        // Only print this if some priority stats were collected
+		if (ddir == DDIR_READ && ts->clat_prio_stat.samples > 0) {
+			sprintf(prio_name, "high prio (%.2f%%) %s",
```

Comment:
This looks a bit free form to me. Won't this be clear from the percentiles?

Reply:
Hopefully, this will be clear from the percentiles, but in case something unexpected happens it seems better to explicitly state which stats are from the high priority reads and which ones are the low priority (although as you mentioned before, these are really regular priority reads, but they are comparatively lower priority). Since this is outputting to the normal human readable format, I thought it was good to be a little bit more verbose. Is there another way to display the data that you'd like to use instead?